apps/meteor/client/views/marketplace/AppsRoute.tsx

```diff
@@ -61,6 +61,8 @@ const AppsRoute = (): ReactElement => {
 		return <PageSkeleton />;
 	}
 
+	console.log('Page: ', page);
+
 	return (
 		<AppsProvider>
 			{(page === 'list' && <AppsPage isMarketplace={isMarketplace} />) ||
```

Comment:
`isMarketplace` is not working properly anymore

Reply:
Solved